snuba/cli/consumer.py

```diff
@@ -54,6 +54,11 @@ def consumer(raw_events_topic, replacements_topic, commit_log_topic, consumer_gr
     dataset_name = dataset
     dataset = get_dataset(dataset_name)
 
+    dataset_brokers = settings.DEFAULT_DATASET_BROKERS
+    bootstrap_server = bootstrap_server or dataset_brokers.get(dataset_name)
+    if not bootstrap_server:
+        bootstrap_server = settings.DEFAULT_BROKERS
```

Comment:
This works (I'm not going to block on it), but I feel like it would be clearer if written:

```python
if bootstrap_server is None:
    if dataset_name in settings.DEFAULT_DATASET_BROKERS:
      bootstrap_server = settings.DEFAULT_DATASET_BROKERS[dataset_name]
    else:
      bootstrap_server = settings.DEFAULT_BROKERS
```

If you don't want to go that far, I think at least not assigning `settings.DEFAULT_DATASET_BROKERS` to an intermediate variable is clearer (I ended up double checking to make sure you weren't intentionally overwriting a local variable for some reason that was used later.)

Reply:
Sure will change

Reply:
done